code-challenges.test.js

```diff
@@ -18,54 +18,165 @@
 
 // a) Create a test with an expect statement using the variable provided.
 
+//setting up test that will take diescribe when shuffleBack is called it will test if the first index is removed  and shuffles the remmaining index in array
+
+
+//??? i tried a few different approches but ended up digging myself a deeper hole with this jest test. 
+//the function works just not the test. 
+
+// describe("when shuffleColors is called", () =>{
+//     it("if argument is colors1, return slice array",() =>{
+//         const argument = "colors1";
```

Comment:
looks like this is passing in a string that happens to be the same name as the array that should be passed into the function